draft-ietf-jsonpath-base.md

```diff
@@ -276,14 +272,18 @@ Unicode Scalar Value:
   In other words, integers in either of the inclusive base 16 ranges 0 to D7FF and
   E000 to 10FFFF.  Both JSON string values and JSONPath queries are sequences of Unicode scalar values.
 
-Singular Query:
-: A JSONPath expression built from segments each of which, regardless of the input value,
-  produces a nodelist containing at most one node.
+Segment:
+: One of the constructs which select children (`[<selectors>]`)
+  or descendants (`..[<selectors>]`) of an input value.
```

Comment:
Note to reviewers: I moved the updated definition of Segment down so that it is just before the definition of Selector. If I had left it in the previous position, I would have felt the need to add a reference from the definition of Segment to the definition of Selector, which I didn't want to get into.